style.css

```diff
@@ -0,0 +1,799 @@
+}
+
+
+.subscribe-footer,
+.subscribe-f{
+    text-align: center;
+    margin-top: 20px;
+    padding: 20px 0;
+    border-top: 2px solid #f4f5f6;
+}
+.subscribe-f .row-title{
+    text-align: left;
+    font-size: 16px;
+    color: #f72210;
+    margin-top: 20px;
+}
+.subscribe-f .image-wrapper{
+    display: inline-block;
+    position: relative;
+    padding-bottom: 13px;
+    padding-right: 90px;
+}
+.subscribe-f .subscribe-footer .image-wrapper>div:last-of-type{
+    position: absolute;
+    top: 13px;
+    left: 80px;
+}
+.image-wrapper{
+    margin-bottom: 15px;
+}
+.subscribe-footer .image-container{
+    width: 170px;
+    height: 228px;
+}
+.subscribe-f .info h2{
+    font-size: 20px;
+    font-weight: 800;
+}
+.subscribe-f .info p{
+    font-size: 15px;
+    margin: 25px 0;
+}
+.subscribe-f .info a{
+    display: inline-block;
+    padding: 10px 10px 10px 12px;
+    color: white;
+    font: bold 18px/18px "Roboto Condensed",sans-serif;
+    text-transform: uppercase;
+    background-color: #f72210;
+}
+.subscribe-footer{
+    padding: 30px 0 30px 0;
+}
+
+.subscribe-footer-news .image-container{
+    background: rgba(0, 0, 0, 0) url("https://g.newsweek.com/themes/newsweek/images/subscribe-img2.gif") no-repeat scroll center center / cover;    height: 160px;
+}
+.subscribe-footer-news .info{
+    border: 1px solid #d9d9d9;
+    border-top: none;
+    padding: 25px 0;
+}
+
+/* subscribe section  */
+
+
+.align-txt{
+    text-align: initial;
+}
+
+
+footer {
+	border-top: 2px solid #f4f5f6;
+	background-color: #fff;
+	padding-top: 30px;
+}
+
+.footer-top {
+	padding-bottom: 20px;
+}
+
+.logo-footer > img {
+	width: 236px;
+	height: 36px;
+	fill: #f72210;
+}
+
+.copyright {
+	vertical-align: bottom;
+	padding-left: 10px;
+}
+
+.media-icons {
+	text-align: right;
+}
+
+.media-icons > i {
+	vertical-align: bottom;
+	font-size: 25px;
+	color: #666;
+	padding-left: 7px;
+}
+
+.menu {
+	display: flex;
+	font-size: 14px;
+	line-height: 15px;
+	list-style: none;
+	margin: 0px;
+	color: #666;
+	padding: 0px;
+}
+
+.leaf {
+	border-right: 1px solid #d9d9d9;
+	padding-right: 10px;
+	margin-right: 10px;
+	margin-bottom: 10px;
+	cursor: pointer;
+}
+
+.leaf:hover {
+	color: #000;
+}
+
+#right {
+	text-align: right;
+	padding: 0px;
+}
+
+.last {
+	border: none;
+}
+
+@media screen and (max-width: 991px) {
+	.logo-img {
+		width: 132px;
+		height: 20px;
+		left: 80px;
+		top: 8px;
+	}
+
+    .image-wrapper{
+        display: block;
+    }
+
+    .subscribe-text{
+        clear:none;
+    }
+	.sticky-navbar {
+		padding: 3px 0px;
+	}
+
+	.nav-date {
+        display: none;
+
+    }
+
+	.sign-in {
+		font-size: 14px;
+		bottom: 0px;
+	}
+
+	.sign-in a {
+		padding: 6px;
+	}
+
+	.header-bottom {
+        display: none;
+	}
+
+	#right-col {
+		margin: auto;
+	}
+
+	.content {
+		padding-top: 40px;
+    }
+
+    .subscribe-footer-news .info{
+        height: 315px;
+    }
+    .subscribe-footer-news .image-container{
+        width: 180%;
```

Comment:
Why do you have `180%` width here :thinking:?

It seems to be making the navbar and the subscribe image non-responsive. See image below where this property is been toggled on chrome web dev tool.

![mag current newsweek com](https://user-images.githubusercontent.com/36057474/62047282-f95f6200-b201-11e9-8a16-0ef40d9bbefe.gif)

Reply:
Thank you so much for the reviews,made changes according to them